app/dashboard/views.py

```diff
@@ -2524,6 +2524,11 @@ def profile_job_opportunity(request, handle):
         profile.save()
     except (ProfileNotFoundException, ProfileHiddenException):
         raise Http404
+    except ValueError:
+        return JsonResponse(
+            {'error': 'Bad request'},
+            status=401
```

Comment:
@thelostone-mc bad request should have status=400

Reply:
udpated